src/main/kotlin/no/nav/faktureringskomponenten/domain/models/Fakturaserie.kt

```diff
@@ -87,20 +87,27 @@ class Fakturaserie(
     }
 
     fun erstattMed(nyFakturaserie: Fakturaserie) {
-        kansellerPlanlagteFakturaer()
+        avbrytPlanlagteFakturaer()
         erstattetMed = nyFakturaserie
         status = FakturaserieStatus.ERSTATTET
     }
 
+    fun kansellerMed(nyFakturaserie: Fakturaserie){
+        avbrytPlanlagteFakturaer()
+        erstattetMed = nyFakturaserie
+        status = FakturaserieStatus.KANSELLERT
+    }
+
     fun bestilteFakturaer(): List<Faktura> {
-        return faktura.filter { it.status != FakturaStatus.OPPRETTET && it.status != FakturaStatus.KANSELLERT }
+        return faktura.filter { it.status == FakturaStatus.BESTILT || it.status == FakturaStatus.MANGLENDE_INNBETALING }
```

Comment:
Hva med  INNE_I_OEBS, og FEIL, er det ikke bestilte fakturaer også?

Reply:
FEIL blir satt på eksternFakturaStatus fra OEBS, men det blir ikke lagt på selve fakturaen
INNE_I_OEBS bruker vi ikke såvidt jeg kan se i koden, men den er utfra en test virker det som om det også er tenkt å ha på eksternFakturaStatus. Selve Faktura objektet vil beholde status == BESTILT

Reply:
Ok, hvor ser du at vi setter MANGLENDE_INNBETALING på Faktura?

Jeg er forvirret av at `Faktura` og `EksternFakturaStatus` deler samme enum for status

Reply:
Denne endringen kan introdusere feil

Reply:
ja den hadde jeg ikke fått med meg. Du mener FEIL og INNE_I_OEBS skal tas med i bestilteFakturaer? må vel teknisk sett det. Vi kan jo ikke kalle de tilbake

Reply:
Akkurat, det jeg tenkte, ja.